docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Live Simulation Mode
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Live Simulation Mode
+
+Forget screen sharing, there's live simulation mode! Let's say you're working on the application logic and want a second opinion or to share your machine with your team mates, live simulation has made this process as easy as sharing a link. It is not necessary to take screenshots anymore because this feature will let all users with the specially generated link to access and discuss in real time the process of your machine.
+
+In order to use this collaborative feature, the user initiating the live simulation must be signed in. When looking at the Stately Studio, you're able to switch between two views: Design mode and Simulate mode.
+
+:::studio
+
+Live simulation is a Pro feature of the Studio. [Check out the features on our Pro plan](/studio-pro-plan.mdx) or [upgrade to try the Pro plan with a 30-day free trial](https://stately.ai/registry/billing).
+
+:::
+
+## Simulate your machine with others
+
+1. Select the **Simulate** mode from the center of the visual editor. When in the **Simulate** mode, you'll see your machine with the initial state turn the color blue. Blue means your machine is not currently being editted or shared, but you are simulating the logic flow of the current machine. In **Simulate** mode, this is where you can go through each state and transition to thoroughly review your logic.
+
+<p>
+<ThemedImage
+  alt="Simulate mode is selected in the center menu and now the machine can not be edited."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-menu.png'),
+    dark: useBaseUrl('/live-sim/live-sim-menu-dm.png'),
+  }}
+/>
+</p>
+
+2. While in **Simulate** mode, you'll notice a button **Live mode** on the bottom left corner. When clicking the **Live mode** button, you'll have the options: 
+- **Join a Live simulation** session
```

Comment:
```suggestion
- **Join a Live simulation** session.
```

Reply:
I wonder if we should split the instructions for Starting and Joining a live simulation into two step-by-step guides with their own headings so we can link to them independently?

Reply:
I can do this in a different PR. Good idea to separate the instructions to be more clear!